frontend/views/components/CopyToClipboard.vue

```diff
@@ -0,0 +1,107 @@
+<template lang='pug'>
+.c-wrapper(@click='copyToClipboard')
+  input.c-invisible-input(
+    type='text'
+    ref='input'
+    :value='textToCopy'
+  )
+  slot
+  transition(name='fade')
+    i18n.c-tooltip(
+      tag='p'
+      v-if='ephemeral.tooltipActive'
+    ) Copied to clipboard!
+</template>
+
+<script>
+import Tooltip from '@components/Tooltip.vue'
```

Comment:
Where do you use the tooltip?

Reply:
it's used at `L32`. for the special invite link. :)
![specialLink](https://user-images.githubusercontent.com/17641213/67146955-dd44fb80-f2ec-11e9-919b-ed205eb493fc.png)